File: src/SOP8/Base/Base.Model/History/SMSReceiver.cs
using System;
using dnsDapperDBUtil.DataAccessLayer.IDAL;

namespace Base.Model.History
{
	public class SMSReceiver : Table
	{
		public enum Fields { sms_hist_sn, rgl_memb_sn };
		public enum WriteFields { sms_hist_sn, rgl_memb_sn };

		public int sms_hist_sn { get; set; }
		public int rgl_memb_sn { get; set; }

		public static string TableName { get { return "his_sms_rcver"; } }

		public override string GetTableName()
		{
			return TableName;
		}

		public override string GetPrimaryCondition()
		{
			return string.Format("{0} = {1} and {2} = {3}", Fields.sms_hist_sn, sms_hist_sn, Fields.rgl_memb_sn, rgl_memb_sn);
		}

		public override Type GetFieldType()
		{
			return typeof(Fields);
		}

		public override Type GetWriteFieldType()
		{
			return typeof(WriteFields);
		}

		public void FromCopy(SMSReceiver obj)
		{
			this.sms_hist_sn = obj.sms_hist_sn;
			this.rgl_memb_sn = obj.rgl_memb_sn;
		}
	}
}

File: src/SOP8/Egovframework/Base/base-model/src/main/java/egovframework/base/model/history/SMSReceiver.java
package egovframework.base.model.history;

import lombok.*;
@Data
@NoArgsConstructor
@AllArgsConstructor
@Builder
public class SMSReceiver {
	public enum Fields { sms_hist_sn, rgl_memb_sn }

	private int sms_hist_sn;

	private int rgl_memb_sn;

	public static final String autoIncreaseField = null;
	public static final String tableName = "his_sms_rcver";
	public static final String[] primaryKeys = { "sms_hist_sn", "rgl_memb_sn" };

}
